cmd/notary/keys.go

```diff
@@ -408,8 +415,8 @@ func (k *keyCommander) importKeys(cmd *cobra.Command, args []string) error {
 	for _, file := range args {
 		from, err := os.OpenFile(file, os.O_RDONLY, notary.PrivKeyPerms)
 		defer from.Close()
-
-		if err = utils.ImportKeys(from, importers); err != nil {
+		passRetriever := k.getRetriever()
```

Comment:
super non-blocking nit: we can remove this line and just directly use `k.getRetriever` in the next line since we don't use the `passRetriever` variable elsewhere